pkg/cluster/k8sres.go

```diff
@@ -716,6 +716,55 @@ func makeResources(cpuRequest, memoryRequest, cpuLimit, memoryLimit string) acid
 	}
 }
 
+func extractPgVersionFromBinPath(binPath string, template string) (string, error) {
+	var pgVersion float32
```

Comment:
Shouldn't it be just an integer?

Reply:
when there can be `9.6`?

Reply:
Yeah, forgot that for a couple of years it still would be a thing.